molecule/ps-innodb-cluster/Jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
     string(name: 'UPSTREAM_VERSION', defaultValue: '8.0.26', description: 'Upstream MySQL version')
     string(name: 'PS_VERSION', defaultValue: '16', description: 'Percona part of version')
     string(name: 'PS_REVISION', defaultValue: '3d64165', description: 'Short git hash for release')
-    choice(name: 'TEST_DIST', choices: ['bento/ubuntu-20.04','bento/ubuntu-18.04','bento/debian-10','bento/debian-9','bento/centos-8','bento/centos-7','generic/oracle8'], description: 'On which image to run test.')
+    choice(name: 'TEST_DIST', choices: ['bento/ubuntu-20.04','bento/ubuntu-18.04','bento/debian-10','bento/debian-9','bento/centos-8','bento/centos-7','bento/oracle-8'], description: 'On which image to run test.')
```

Comment:
Why do we leave centos-8?

Reply:
I will remove Deb-9 and Centos-8 when i create a PR to add Oracle Linux 9 and Ubuntu Jammy

Reply:
Removed Deb-9 and Centos-8

Reply:
Names are now in sync also